### DISCUSSION TITLE
Unable to connect to external container

### DISCUSSION THREAD

**Discussion by Recho1235** [qdm12/gluetun#1049] [Q&A]
Hi all,

I'm attempting to run my containers through gluetun by using network container gluetun and exposing the required ports for those contains within gluetun itself as per the documentation. Once doing this it looks like my containers deploy with no problems but I am unable to access my external containers that run through gluetun using the usual localhost:port. Does anyone know why this is and how to fix it?

Thank you

**Comment by qdm12** [comment]
First sorry for the big delay.

Please share your configuration (remove credentials), since I can't really help otherwise. You can also try just pulling the latest image and check if it's fixed since then 🤔
